task_8/toma.lobanova/homework8/src/main/java/ua/od/hillel/human/Human.java

```diff
@@ -0,0 +1,80 @@
+package ua.od.hillel.human;
+
+import java.time.DateTimeException;
+import java.time.LocalDate;
+import java.util.Calendar;
+import java.util.GregorianCalendar;
+
+public class Human {
+    String name = "unspecified";
+    String surname = "unspecified";
+    GregorianCalendar birthday = new GregorianCalendar(1975,
+            Calendar.DECEMBER, 31);
+    boolean married = false;
```

Comment:
They are false by default during the object creation. No need to specify it directly.